packages/client/katello-host-tools/katello-host-tools.spec

```
@@ -234,6 +234,13 @@ rm %{buildroot}%{_sbindir}/katello-tracer-upload
 rm %{buildroot}%{plugins_confdir}/tracer_upload.conf
 %endif
 
+%if %{yum_install}
```

Shouldn't this be something like:
```suggestion
%if !%{zypper_install}
```

Or is that not supported?

---

Thanks for your suggestion. I tested it and it works.